samples/model/README.md

```diff
@@ -0,0 +1,49 @@
+# Model Sample
+
+This sample demonstrates how to start models/create a model using your own model script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do).
+
+## Preparing the Model Script
+
+### Get an Image
+
+All SageMaker models are run from within a container with all necessary dependencies and modules pre-installed and with the model scripts referencing the acceptable input and output directories. Sample container images are [available](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-algo-docker-registry-paths.html).
+
+A container image URL and tag looks has the following structure:
+```
+<account number>.dkr.ecr.<region>.amazonaws.com/<image name>:<tag>
+```
+
+### Updating the Model Specification
+
+In the `my-model.yaml` file, modify the placeholder values with those associated with your account and model.
+
+# Submitting your Model
+
+## Create your Model
+To submit your prepared model specification, apply the specification to your EKS cluster as such:
+```
+$ kubectl apply -f my-model.yaml
+models.sagemaker.services.k8s.aws.amazon.com/my-model created
+```
+
+## List your Model
+To monitor the model status, you can use the following command:
```

Comment:
Can we reword this text to change the word `monitor`? and list resources rather than a single resource

 List your Model`s`
Use the following command to list all [models/jobs/endpoints etc.] created using the ACK controller:


kubectl get models.sagemaker.services.k8s.aws ~~my-model~~


because list means multiple in general. for singular user, can run describe instead. Same for other resources